Bottom: c40d5f75c2de71fbbab1c6019f2231c71f041352
Top:    8b13ef364cc34849d708477688bc5cb08a6e1ad9
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-17 08:54:51 -0500

Create certificate directories during image build



---
diff --git a/docker/docker-mantisbt/configure.sh b/docker/docker-mantisbt/configure.sh
index 711b554f..a8e08843 100755
--- a/docker/docker-mantisbt/configure.sh
+++ b/docker/docker-mantisbt/configure.sh
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 
+# Create certificate directories
+mkdir -p /etc/grid-security/certificates
+mkdir -p /etc/ssl/certs/
+mkdir -p /etc/ssl/private
+
 # comment out apache2 config file lines that reference the environment variables
 sed -i 's|^Mutex file|#Mutex file|' /etc/apache2/apache2.conf
 sed -i 's|^PidFile |#PidFile |' /etc/apache2/apache2.conf
